content/en/docs/reference/ports-and-protocols.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Ports and Protocols
+content_type: concept
+weight: 50
+---
+
+When running Kubernetes in an environment with strict network boundaries, such 
+as on-premises datacenter with physical network firewalls or Virtual 
+Networks in Public Cloud, it is useful to be aware of the ports and protocols 
+used by Kubernetes components
+
+## Control-plane node(s)
+
+| Protocol | Direction | Port Range | Purpose                 | Used By                   |
+|----------|-----------|------------|-------------------------|---------------------------|
+| TCP      | Inbound   | 6443*      | Kubernetes API server   | All                       |
+| TCP      | Inbound   | 2379-2380  | etcd server client API  | kube-apiserver, etcd      |
+| TCP      | Inbound   | 10250      | Kubelet API             | Self, Control plane       |
+| TCP      | Inbound   | 10259      | kube-scheduler          | Self                      |
+| TCP      | Inbound   | 10257      | kube-controller-manager | Self                      |
+
+## Worker node(s)
+
+| Protocol | Direction | Port Range  | Purpose               | Used By                 |
+|----------|-----------|-------------|-----------------------|-------------------------|
+| TCP      | Inbound   | 10250       | Kubelet API           | Self, Control plane     |
+| TCP      | Inbound   | 30000-32767 | NodePort Services†    | All                     |
+
+† Default port range for [NodePort Services](/docs/concepts/services-networking/service/).
+
+Any port numbers marked with * are overridable, so you will need to ensure any
+custom ports you provide are also open.
+
+Although etcd ports are included in control-plane nodes, you can also host your own
+etcd cluster externally or on custom ports.
```

Comment:
- Maybe put that before the “Worker node(s)” heading?
- optionally: hyperlink to `/docs/setup/production-environment/tools/kubeadm/ha-topology/#external-etcd-topology`

Reply:
We could also put that comment in the “Check required ports” section of the kubeadm setup page. Your call I think.

Reply:
Good call on placement. Moved under control plane section